langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -39,6 +44,20 @@ public isolated function length(map<any|error> m) returns int =@java:Method {
 
 # Returns an iterator over a map.
 #
+# ```ballerina
+# map<string> vowels = {a: "A", e: "E", i: "I", o: "O", u: "U"};
+#
+# // iterator() returns an iterator object with the `next()` method.
+# var vowelsIter = vowels.iterator();
+#
+# // `next()` can be used to get the next element of the map.
+# record {|string value;|}? nextVal = vowelsIter.next();
+#
+# while nextVal != null {
+#     nextVal = vowelsIter.next();
```

Comment:
Do we need this part in the langlib as examples

Reply:
I also think we can remove this.

Reply:
Yeah, we can remove this. I thought of delivering a better a use-case for `next()` too